app/javascript/src/components/Dashboard/Contacts/index.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from "react";
+
+import { Button, Table } from "neetoui";
+import { Container, Header } from "neetoui/layouts";
+
+import { CONTACTS_TABLE_COLUMNS, CONTACTS_TABLE_ROWS } from "./constants.jsx";
+
+import SideMenuBar from "../SideMenuBar";
+
+const Contacts = () => {
+  const [showMenu, setShowMenu] = useState(true);
+
+  return (
+    <>
+      <SideMenuBar showMenu={showMenu} title="Contacts" />
+      <Container>
+        <Header
+          menuBarToggle={() => setShowMenu(!showMenu)}
```

Comment:
Go through the Slack thread very carefully.

Reply:
Sorry, I've gone through the slack thread again but I couldn't get the mistake in these lines.
Can you please point it out @yedhink ?

Reply:
@allenjiji https://academy.bigbinary.com/learn-react/pages/know-when-to-use-previous-state-for-state-update